recipes/jaws/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.2.3" %}
+
+package:
+  name: jaws
+  version: {{ version }}
+
+source:
+  url: https://github.com/jaws/jaws/archive/{{ version }}.tar.gz
+  sha256: b8cd6e76f01a3786d32880c757b02aa82651eb9fffeb0bc8aa155dac02212b38
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pandas
+    - numpy >=1.11
+    - xarray
+    - pytz
+    - netcdf4
+    - matplotlib
```

Comment:
Are all of these build requirements?

Reply:
These were only run-time, removed from build requirements.